src/distributed_ls/Euclid/Euclid_dh.c

```diff
@@ -70,7 +70,7 @@ void Euclid_dhCreate(Euclid_dh *ctxOUT)
   strcpy(ctx->krylovMethod, "bicgstab");
   ctx->maxIts = 200;
   ctx->rtol = 1e-5;
-  ctx->atol = _ATOL_;
+  ctx->atol = 1.0e-50;
```

Comment:
What's wrong with the `#define _ATOL_` ?

Reply:
ATOL was being defined only in the `_hypre_Euclid.h` header file. Thus, running the `headers` script would not include it in `_hypre_Euclid.h`. Since ATOL is being used in this single place, I decided to erase it and assign `ctx->atol` to the value instead. As a matter of fact, I ended up missing that ATOL can assume 1e-16 when HYPRE_SINGLE is defined. I'm fixing this with commit f7673f1